src/sentry/rules/processing/delayed_processing.py

```diff
@@ -189,7 +190,7 @@ def get_condition_group_results(
         # The comparison interval is only set for the second query of a percent
         # comparison condition.
         comparison_interval = (
-            condition_inst.intervals[unique_condition.comparison_interval][1]
+            COMPARISON_INTERVALS[unique_condition.comparison_interval][1]
```

Comment:
Thoughts on making this a bit more resilient and using `.get` to access the dictionary attribute? that way we could provide a default value  here as well.